examples/logit_ratio.py

```diff
@@ -160,8 +162,14 @@ def sample_coefs(cmodel: Cascade) -> pd.DataFrame:
 
 def main():
     # load data
-    df = pd.read_csv(data_path)
-    df = df[df.include].reset_index(drop=True)
+    df_all = pd.read_csv(data_path)
+    df_all = df_all[df_all.include].reset_index(drop=True)
+
+    national_index = df_all.ihme_loc_id.str.len() == 3
+    df_national = df_all[national_index].reset_index(drop=True)
+    df_subnational = df_all[~national_index].reset_index(drop=True)
+
+    df = df_national
```

Comment:
Uses only national data for global, super-region, region computation. If we want to use all data replace this by `df = df_all`.